README.md

```diff
@@ -21,4 +21,13 @@ Make company global travel and accommodation easy and convenient for the strong
 ### Setting up sequelize to work:
 - Install `sequelize` and `sequelize-cli` globally
 - For development we are using a `DEV_DATABASE_URL` variable to keep our DB url in `.env` (Add `export` in front of it)
-- Then run `npm run db-migrate:dev` to run migrations
\ No newline at end of file
+- Then run `npm run db-migrate:dev` to run migrations
+
+### Running Application on Docker
+- Install Docker from [Docker For Mac](https://docs.docker.com/docker-for-mac/install/) or [Docker For Windows](https://docs.docker.com/docker-for-windows/install/)for windows you will also need to install [docker-compose](https://docs.docker.com/compose/install/)
+- After cloning the application from Github. Switch to the main app where there's the Dockerfile and the docker-compose.yml file
+- Then run `docker build -t barefoot-img .`  To create an image of the application. barefoot-img is the name of our application image
+- Then run `docker run -p barefoot-img` to confirm the image is created successfully and is running. You should get a message Listening to port 3000. We need to map the port which the app is running to an empty port on our system.
```

Comment:
since we don't expose a port in our Dockerfile i believe `docker run -p barefoot-img` won't run the image. We only need the `docker-compose up` to both build the image and run it

Reply:
Nice point out. I have updated the read me.